### DISCUSSION TITLE
Does ND support grabbing album art from the songs?

### DISCUSSION THREAD

**Discussion by gerroon** [navidrome/navidrome#2615] [Q&A] [answered]
Hi

What kind of album art stuff supported in ND? For instance does it only support something like "cover.jp" in the folder? Does it support art in the songs if no album art is in the folder? 

I am seeing alot of covers missing in ND compared to Gonic or Airsonic etc so I am wondering why that is the case.

**Comment by metalheim** [comment]
Navidrome supports both embedded artwork aswell as files in the album folder. 
What navidrome chooses is configurable, please refer to the documentation:
https://www.navidrome.org/docs/usage/artwork/

**Reply by gerroon** [reply]
Thanks for the answer. According to the docs I can use "ND_CONFIGFILE=/data/navidrome.toml" config for Docker. My question is, does this path refers to a path inside the container or outside of the container (ie the actual file system)?

**Comment by metalheim** [comment, marked as answer]
> Thanks for the answer. According to the docs I can use "ND_CONFIGFILE=/data/navidrome.toml" config for Docker. My question is, does this path refers to a path inside the container or outside of the container (ie the actual file system)?

If youre running docker, everything always refers to inside the Container. That's the point of containers. 

You probably have the /data directory mounted from the host to the container already, so you can put the config file on the host and its mounted to /data inside the container.
